src/python/zigopt/authentication/login.py

```diff
@@ -13,13 +13,11 @@
 from zigopt.user.model import User, do_password_hash_work_factor_update, password_matches
 
 
-@deal.has()
```

Comment:
unrelated changes? fine to remove if needed, but seems separate

Reply:
Yeah the API was breaking locally because of a contract error